gpt_index/readers/download.py

```diff
@@ -0,0 +1,67 @@
+"""Download loader from the Loader Hub."""
+
+import json
+import os
+import subprocess
+import sys
+from importlib import util
+
+from langchain.utilities import RequestsWrapper
+
+from gpt_index.readers.base import BaseReader
+
+LOADER_HUB_URL = (
+    "https://raw.githubusercontent.com/emptycrown/loader-hub/main/loader_hub"
+)
+
+
+def download_loader(loaderClassName: str) -> BaseReader:
```

Comment:
nit: arg name should be snake case e.g. loader_class

Reply:
true -- switching context between js and python had weird effect on your brain